### PR TITLE
Update tech-insight BE README.

### PR DESCRIPTION
**Pull request by Xantier**
Ref https://github.com/backstage/backstage/pull/8845/files#r789669414

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [n/a] Tests for new functionality and regression tests for bug fixes
- [n/a] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/tech-insights-backend/README.md [hunk 1]
```diff
@@ -85,10 +85,10 @@ const myFactRetriever = {
    */
 };
 
-const myFactRetrieverRegistration = createFactRetrieverRegistration(
-  '1 * 3 * * ', // On the first minute of the third day of the month
-  myFactRetriever,
-);
+const myFactRetrieverRegistration = createFactRetrieverRegistration({
+  cadende: '1 * 3 * * ', // On the first minute of the third day of the month
```
Comment (julioz) [inline plugins/tech-insights-backend/README.md:89]:
```suggestion
  cadence: '1 * 3 * * ', // On the first minute of the third day of the month
```

**Review by Rugvip — APPROVED** [review]

**Comment by Xantier** [comment]
Getting that DCO to pass is the most difficult thing in the world :D
Note to self: never click that 'commit suggestion' button ever again.

**Comment by jhaals** [comment]
> Getting that DCO to pass is the most difficult thing in the world :D Note to self: never click that 'commit suggestion' button ever again.

haha it can be tricky sometimes! If you do use the UI make sure to add the `Signed-off-by` to the commit line :) 

This branch fails due to a previous issue we had in master with changesets. If you rebase your branch it should be good to go.

**Comment by Xantier** [comment]
Updated @jhaals 

> haha it can be tricky sometimes! If you do use the UI make sure to add the Signed-off-by to the commit line :)

Did that actually but that didn't bring any alleviation to the pain.

**Pull request merged**